spec/requests/disability_compensation_form_request_spec.rb

```diff
@@ -246,6 +246,32 @@ def test_error(cassette_path, status, headers)
           expect(form.dig('form526', 'form526', 'toxicExposure')).not_to eq(nil)
         end
 
+        it 'sets claims_api as submit_endpoint when includes_toxic_exposure query param value is true' do
```

Comment:
Nit: I'd probably do this with context blocks for clarity:

```ruby
context 'when the includes_toxic_exposure query param value is true' do
  let(:query_param) { '?includes_toxic_exposure=true' }

  it 'sets claims_api as submit_endpoint' do
    #...
  end
end
```

```

Reply:
Implemented per suggestion.